src/main/java/Duke.java

```diff
@@ -1,10 +1,37 @@
+import java.util.Objects;
+import java.util.Scanner;
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner reader = new Scanner(System.in);
+        Task[] items = new Task[100];
+        int counter = 0;
+        System.out.println("Hi, I'm bob");
+        System.out.println("What's up");
+        String word = reader.nextLine();
+        while (!word.equals("bye")) {
+            if (word.equals("list") && counter > 0) {
+                for (int i = 1; i <= counter; i++) {
+                    System.out.println(i + ".[" + items[i-1].getStatusIcon() + "] " + items[i-1].description);
+                }
+            } else if (word.contains("mark")) {
+                String[] markArgs = word.split(" ");
+                int taskNo = Integer.parseInt(markArgs[1]);
+                if (markArgs[0].equals("unmark")) {
+                    items[taskNo-1].unmarkAsDone();
+                    System.out.println("This task is not done: " + "[" + items[taskNo-1].getStatusIcon() +
+                        "] " + items[taskNo-1].description);
+                } else {
+                    items[taskNo-1].markAsDone();
+                    System.out.println("This task is done: " + "[" + items[taskNo-1].getStatusIcon() + "] " +
+                        items[taskNo-1].description);
```

Comment:
Indentation for wrapped lines should be 8 spaces (i.e. twice the normal indentation of 4 spaces) more than the parent line. For example,

```Java
setText("Long line split"
        + "into two parts.");
```